docs/usage.md

```diff
@@ -383,17 +383,21 @@ Turn this on to utilize BWA Mem instead of `bwa aln` for alignment. Can be quite
 
 Users can configure to keep/discard/extract certain groups of reads efficiently in the nf-core/eager pipeline. 
 
-### `--bam_keep_mapped_only`
+### `--bam_analyse_mapped_only`
 
-This can be used to only keep mapped reads for downstream analysis. By default turned off, all reads are kept in the BAM file. Unmapped reads are stored both in BAM and FastQ format e.g. for different downstream processing.
+This can be used to only keep mapped reads in the BAM file for downstream analysis. By default turned off, where all reads are kept in the bam file. Unmapped reads are stored both in BAM and FastQ format e.g. for different downstream processing.
```

Comment:
Possibly rephrase again for clarity (my bad): 
Use only mapped reads in the BAM file for downstream analysis. Unmapped reads are stored in a separate BAM and FASTQ format e.g. for different downstream processing. By default turned off, where default behaviour is to keep both mapped and unmapped reads in the output BAM file.

Reply:
I wonder if it makes sense to instead have a independent flag to save unmapped reads in FASTQ rather than BAM format. In this case we will be make redundancy.